tests/subsys/fs/multi-fs/src/nffs_test_utils.c

```diff
@@ -219,7 +219,10 @@ void nffs_test_util_create_file_blocks(const char *filename,
 	int i;
 
 	/* We do not have 'truncate' flag in fs_open, so unlink here instead */
-	fs_unlink(filename);
+	rc = fs_unlink(filename);
```

Comment:
Right vales are {0, `-ENOENT`}. Worth to check exactly that as other errcode  (than `-EINVA`) are also possible to be returned.

Reply:
Hii, as per current fs_unlink() can return, 0, -ENOENT, or -EINVAL
In this case , i was only checking -EINVAL, as if filename or path is incorrect, then fail

Anyway i modified to only check for true cases 0 or -ENOENT, pls check
Pls check.